projects/win32/numpy.cmake

```diff
@@ -1,3 +1,3 @@
-set(numpy_platform_string "cp36-none-win_amd64")
+set(numpy_platform_string "cp37-none-win_amd64")
```

Comment:
@cryos It looks like for Windows, the name of the file inside the ` numpy-1.16.3-python3.7.tar.gz` file is actually `...cp37-cp73m...` rather than `...cp37-none...`.

I wonder if I should change this to `cp37-cp37m`, or change the name of the file? I can't remember if I named that file myself or if it was named that automatically.

Reply:
Actually, if I'm remembering it correctly now, I think I just downloaded the files from [here](https://pypi.org/project/numpy/#history). So that name changed and I must have missed it. I'll add a PR and start looking at the issues on mac and linux.